MiniApp/Classes/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -174,6 +184,18 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
     }
 }
 
+extension MiniAppScriptMessageHandler: MiniAppMessageDelegate {
+    func getUniqueId() -> String {
+        return ""
+    }
+
+    func requestPermission(permissionType: MiniAppPermissionType, completionHandler: @escaping (Result<MASDKPermissionResponse, MASDKPermissionError>) -> Void) {
+    }
+
+    func requestCustomPermissions(permissions: [MASDKCustomPermissionModel], completionHandler: @escaping (Result<[MASDKCustomPermissionModel], MASDKCustomPermissionError>) -> Void) {
+    }
+}
+
```

Comment:
extension is not `public`, which means the host app will get an error if it failed to conform to any `MiniAppMessageDelegate` methods. Is it the real intention here ?

Reply:
that's right, this is just for internal use. I implemented this because I wanted a fallback for this : `weak var hostAppMessageDelegate: MiniAppMessageDelegate?`
The weird think is that this protocol is optional, but you're right in your next comment, it also is a mandatory parameter, so maybe we can assume it will always be there and remove the optional

Reply:
I made it simpler, just assuming that there should not be an internal use of `MiniAppMessageDelegate`